README.md

```diff
@@ -88,9 +88,8 @@ You can learn more about using Cornucopia by reading our [book](https://cornucop
 The [book](https://cornucopia-rs.netlify.app/book/index.html) is the place to go to get more in-depth explanations, but here is the simplest of tasters to give you an idea.
 
 Let's say you have the following PostgreSQL queries
+**/queries/some_query_file.sql**
```

Comment:
I think we should add "located in", as in "Let's say you have the following PostgreSQL queries located in ...".

Also, the path should be monospaced `like this` and not bolded **like that**, in my opinion (might be wrong though, I don't know much about typography).

Finally the leading slash (`/`) in the path suggests that this is an absolute path, while it really should be relative (to the cargo crate root).